test/signale-tests.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
```

Comment:
This file should be removed now that it's ignored.

Reply:
removed this one also